vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericWaitSteps.java

```diff
@@ -90,6 +93,45 @@ private <T> void waitForConditionValidatingVisibility(Locator locator,
         waitForCondition(conditionFactory.apply(locator));
     }
 
+    /**
+     * Waits for expected number of elements.
+     * @param locator The locator to find an element.
+     * @param comparisonRule The rule to match the variable value. The supported rules:
+     *                       <ul>
+     *                       <li>less than (&lt;)</li>
+     *                       <li>less than or equal to (&lt;=)</li>
+     *                       <li>greater than (&gt;)</li>
+     *                       <li>greater than or equal to (&gt;=)</li>
+     *                       <li>equal to (=)</li>
+     *                       <li>not equal to (!=)</li>
+     *                       </ul>
+     * @param expectedNumberOfElements
```

Comment:
missing description, also can we use the same param name across wording and method?